crates/rpc-types-eth/src/block.rs

```diff
@@ -149,6 +149,60 @@ impl Header {
     }
 }
 
+impl TryFrom<Header> for alloy_consensus::Header {
+    type Error = String;
```

Comment:
This should be `ConversionError`

Reply:
done. i just added a "::Custom" error, since the ConversionError seems to be made for Transactions, i didnt want to add other variants to it